tests/cli/test_sysctl_init.py

```diff
@@ -0,0 +1,160 @@
+from pathlib import Path
+from unittest.mock import Mock, mock_open, patch
+
+from hathor.builder.sysctl_builder import SysctlBuilder
+from hathor.cli.run_node import RunNode
+from hathor.utils import txt
+from tests import unittest
+
+file_p2p_config = {
+    'p2p.max_enabled_sync': 7,
+    'p2p.rate_limit.global.send_tips': (5, 3),
+    'p2p.sync_update_interval': 17,
+    'p2p.always_enable_sync': ['peer-1', 'peer-2'],
+}
+
+file_p2p_valid_data = '''\
+p2p.max_enabled_sync=10
+p2p.rate_limit.global.send_tips=2,1
+p2p.sync_update_interval=15
+p2p.always_enable_sync=["peer-1","peer-2"]
+p2p.always_enable_sync.readtxt="/hathor/conf/enablesync.txt"\
+'''
+
+file_invalid_property = '''\
+invalid.property=10\
+'''
+
+file_invalid_value = '''\
+p2p.rate_limit.global.send_tips=!!tuple [1,2]\
+'''
+
+
+class SysctlInitTest(unittest.TestCase):
+
+    @patch('builtins.open', mock_open(read_data=file_p2p_valid_data))
```

Comment:
To prevent patching `builtins.open` and `os.path.isfile`, you can simply create a temporary folder, put a file there, test your code, and delete the folder. We do it in many other tests that you can use as a reference.